src/grisp_gpio.erl

```diff
@@ -47,17 +92,48 @@ configure_slot(gpio2, {T1, T2, T3, T4}) ->
     configure(gpio2_3, T3),
     configure(gpio2_4, T4),
     ok;
+configure_slot(jumper, {J1, J2, J3, J4, J5}) ->
+    configure(jumper_1, J1),
+    configure(jumper_2, J2),
+    configure(jumper_3, J3),
+    configure(jumper_4, J4),
+    configure(jumper_5, J5),
+    ok;
 configure_slot(spi1, disable_cs) -> configure(ss1, output_1);
 configure_slot(spi1, enable_cs)  -> configure(ss1, periph_c);
 configure_slot(spi2, disable_cs) -> configure(ss2, output_1);
 configure_slot(spi2, enable_cs)  -> configure(ss2, periph_c).
 
-get(Pin) -> gen_server:call(?MODULE, {command, <<(index(Pin)):8, 2:8>>}).
+start_polling() ->
+    Command = <<?CMD_START_POLLING:8>>,
+    gen_server:call(?MODULE, {command, Command, undefined}).
+
+stop_polling() ->
+    Command = <<?CMD_STOP_POLLING:8>>,
+    gen_server:call(?MODULE, {command, Command, undefined}).
+
+get(Pin) ->
+    Command = <<?CMD_PIN_GET:8, (pin_index(Pin)):8>>,
+    gen_server:call(?MODULE, {command, Command, fun bin_to_bool/1}).
```

Comment:
Shouldn't the `PostFun` be executed in the context of the caller since it only transforms the results? We get greater concurrency this way.